aries-site/src/pages/components/layer/center-layer.mdx

```diff
@@ -0,0 +1,68 @@
+import { Notification } from 'grommet';
+import { Example } from '../../../layouts';
+import { CenterLayerAnatomy, ContentDrivenLayout } from '../../../examples';
+
+## Anatomy
+
+This diagram highlights details specific to center layers. See [general layer anatomy](/components/layer#anatomy) for specifications applying to all layers.
+
+<Example plain>
+  <CenterLayerAnatomy />
+</Example>
+
+| Label | Region             | Purpose                                                          | Required?                                  | Notes                                                                                                                                                                                                                                |
+| ----- | ------------------ | ---------------------------------------------------------------- | ------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **1** | **Close button**   | Closes the layer.                                                | Required on layers with no footer actions. | Use "FormClose" icon.                                                                                                                                                                                                                |
+| **2** | **Footer actions** | Allows the user to confirm, apply, or cancel the layer's action. | Required on actionable layers.             | The actions should be right-aligned with primary action on the right. Right-alignment signals progress and forward momentum. One action should be a "Cancel" button that closes the layer, returning the user to the previous state. |
+
+See [layer closing behavior](/components/layer#layer-closing-behavior) for details.
+
+<Notification
+  status="info"
+  message="A layer should either have actions in the footer or a close button in the header, not both."
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+## Examples
+
+### Destructive confirmation
+
+<Notification
+  status="info"
+  message="Additional destructive confirmation patterns differing based off of level of severity are coming soon."
+  width="large"
+  margin={{ bottom: 'medium' }}
+/>
+
+For highly destructive actions, have the user type a string that matches the name or path of the object to be deleted. This approach ensures the user is aware of the consequences of the action.
```

Comment:
Is "highly destructive" clear enough to indicate when to follow this pattern? I see the note above about "coming soon", but I wonder if we might be confusing things by adding only this example of destructive behavior as opposed to waiting for the additional patterns to provide clearer guidance on "which" "when".

Reply:
We haven't done the design work yet on what the threshold is to require this pattern. I am happy to remove if we want to wait until we have more patterns.